db/migrate/20230523090028_remove_email_on_proposal_notification_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveEmailOnProposalNotificationFromUsers < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :users, :email_on_proposal_notification, :boolean
```

Comment:
```suggestion
    remove_column :users, :email_on_proposal_notification, :boolean, default: true
```

This way running this migration and then reverting it will result in the same properties being applied to this  field.